docker/scripts/osie.sh

```diff
@@ -370,7 +370,10 @@ EOF
 			  Ec2:
 			    timeout: 60
 			    max_wait: 120
-			    metadata_urls: [ 'http://metadata.packet.net', 'http://147.75.207.1:80' ]
+			    metadata_urls:
```

Comment:
The empty line before the list of URL's is suspicious -- is it intentional?

Reply:
metadata_urls is an array. I've changed from `[]` syntax to `lines prefixed with -` instead.